test/components/email_link_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'component_test_helper'
+
+class EmailLinkTest < ComponentTestCase
+  def component_name
+    "email-link"
+  end
+
+  test "renders nothing without a href" do
```

Comment:
We need to add a test that nothing is rendered when nothing is passed to the component too